src/app/data/services/i18n.ts

```diff
@@ -1,6 +1,8 @@
 class I18n {
   private static translations:{[key:string]:string} = {
-    'cms.edit.message.conflict': 'Could not save the content. A newer Version already exists on the server.'
+    'cms.edit.message.conflict': 'Could not save the content. A newer Version already exists on the server.',
+    'cms.edit.message.unauthorized': 'You are not authorized to change the content of this block.',
+    'cms.edit.message.no-connection': 'No connection to the server. Either you are offline or the server is down.'
```

Comment:
I would remove the second sentence as there are a lot of other things that could happen to the network traffic.